src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -381,8 +381,9 @@ public enum SyntaxKind : ushort
         AndKeyword = 8439,
         /// <summary>Represents <see langword="not"/>.</summary>
         NotKeyword = 8440,
-        /// <summary>Represents <see langword="data"/>.</summary>
-        DataKeyword = 8441,
+
+        // 8441 open for future use.
```

Comment:
>// 8441 open for future use. [](http://example.com/codeflow?start=8&length=28)

@333fred, @jcouv

I believe the decision of compat council was to reserve the number. I.e. prevent its reuse now or in the future. It doesn't look like this comment is in line with that.